oxo.py

```diff
@@ -0,0 +1,133 @@
+class OxoBoard:
+    def __init__(self, x, y):
+
+        self.Board = {(0, 0): 0, (1, 0):0, (2, 0):0, (0, 1):0, (1, 1):0, (2, 1):0, (0, 2):0, (1, 2):0, (2, 2):0}
+
+    def get_square(self, x, y):
+
+        return self.Board[x,y]
+
+    def set_square(self, x, y, mark):
+
+        if self.Board[x, y] == 0:
+            self.Board[x, y] = mark
+            return True
+
+        else:
+            return False
+
+    def is_board_full(self):
+
+        SquareSearch = 0
+        for x in xrange(0, 3):
+
+            for y in xrange(0, 3):
+
+                if self.Board[x, y] > 0:
+                    SquareSearch += 1
+
+                if SquareSearch == 9:
+                    return True
+
+                elif [x, y] == (2, 2):
+                    return False
+
+    def get_winner(self):
+
+        if self.Board[1,1] !=0:
+
+            if self.Board[0,0] == self.Board[1,1]:
+
+                if self.Board [2,2] == self.Board[1,1]:
+                    return current_player

```

Comment:
`current_player` is a global variable -- this happens to work when `oxo.py` is run as a program, but not from the `oxotest.py` script. You should return e.g. `self.Board[1,1]` instead here.